schema.sql

```diff
@@ -0,0 +1,79 @@
+/* CREATE DATABASE */
+
+CREATE DATABASE catalog_app;
+
+-- Enter into the database
+\c catalog_app;
+
+-- Create the tables --
+
+-- Create genre table
+CREATE TABLE Genre(
+  id      INT PRIMARY KEY GENERATED ALWAYS AS IDENTITY,
+  name    varchar(50),
+);
+
+/* Create Authors table */
+CREATE TABLE Author(
+  id      INT GENERATED ALWAYS AS IDENTITY,
+  first_name    varchar(50),
+  last_name     varchar(50),
+  items         varchar(50)[],
+  PRIMARY KEY(id)
+);
+
+-- Create items table (parent table)
+CREATE TABLE Item(
+  id INT PRIMARY KEY GENERATED ALWAYS AS IDENTITY,
+  genre_id INT REFERENCES Genre(id),
+  author_id INT REFERENCES Author(id),
+  label_id INT REFERENCES Label(id),
+  publish_date  DATE,
+  archived BOOLEAN,
+);
+
+/* Create Games table (child table of item) */
+CREATE TABLE Games(
+  id      INT GENERATED ALWAYS AS IDENTITY,
+  genre   varchar(50),
+  author  varchar(50),
+  label   varchar(50),
+  publish_date  date,
+  archived  bool,
+  multiplayer bool,
+  last_played_at  date,
+  author_id   INT REFERENCES Author(id),
+  genre_id INT REFERENCES Genre(id),
+  label_id INT REFERENCES Label(id),
+  PRIMARY KEY(id)
+);
+
+-- Create music album table (child table of item)
+CREATE TABLE MusicAlbum(
+  id INT PRIMARY KEY GENERATED ALWAYS AS IDENTITY,
+  item_id INT REFERENCES Item(id),
+  on_spotify BOOLEAN
+);
+
+-- Create labels table
+
+CREATE TABLE Label(
+  id INT, 
+  title VARCHAR(255), 
+  color VARCHAR(255), 
+  PRIMARY KEY(id)
+);
+-- Create books table
+
+CREATE TABLE Books(
+    id INT, 
+    name VARCHAR(255), 
+    published_date DATE, 
+    publisher VARCHAR(255), 
+    cover_state VARCHAR(255), 
+    archived BOOLEAN, 
+    author_id INT REFERENCES Author(id), 
+    label_id INT REFERENCES Label(id), 
+    genre_id INT REFERENCES Genre(id), 
```

Comment:
Great job team 🤝

- [ ] As stated in the project requirements, kindly ensure all foreign keys columns have corresponding indexes.

Reply:
#### Hi @1GORDON 👋 
You are right, we will fix that.
Thank you for your feedback and your time in the review. ✨